.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 *.test
 .envrc
+pctl
```

Comment:
so I totally had this before and it git ignored the `cmd/pctl` directory, double check its not done that again?

Reply:
what do you mean?

Reply:
all good, no idea what you did

Reply:
me neither.